src/types/Color.ts

```diff
@@ -0,0 +1,80 @@
+class RGBA {
+	r: number
+	g: number
+	b: number
+	a: number
+}
+
+class Color implements SBVRType<number, RGBA> {
+	types = {
+		  postgres: 'INTEGER'
+		, mysql: 'INTEGER'
+		, websql: 'INTEGER'
+		, odata: {
+			  name: 'Self.Color'
+			, complexType: `<ComplexType Name="Color">
+											 <Property Name="r" Nullable="false" Type="Edm.Int8"/>
```

Comment:
Really strange indents. I also recommend putting the closing backtick on its own line